trace_decoder/src/decoding.rs

```diff
@@ -750,7 +750,7 @@ fn create_minimal_state_partial_trie(
 // trie somewhere else! This is a big hack!
 fn create_minimal_storage_partial_tries<'a>(
     storage_tries: &HashMap<HashedAccountAddr, HashedPartialTrie>,
-    state_accounts_with_no_accesses_but_storage_tries: &HashMap<HashedAccountAddr, TrieRootHash>,
+    _state_accounts_with_no_accesses_but_storage_tries: &HashMap<HashedAccountAddr, TrieRootHash>,
```

Comment:
we should probably remove this argument from the function instead of keeping it unused

Reply:
FWIW I have a kanban task to actually remove this method. It is much better to do this at the start of each txn we process instead of when creating the partial trie.